HW16/script.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+ 
+A=$1
+B=$2
+C=$3
+
+if [ -z $A ] || [ -z $B ] || [ -z $C ] || [[ ! $A == ?(-)+([0-9]) ]] || [[ ! $B == ?(-)+([0-9]) ]] || [[ ! $C == ?(-)+([0-9]) ]]; then
+    echo "The value of A, B, or C is empty or is not a number" 
+elif (( $A == $B )); then
```

Comment:
To make your code smaller create a function to compare two variables. Build your output based on function outputs.

Reply:
Done
![image](https://user-images.githubusercontent.com/44346029/115994428-97eb4380-a5df-11eb-8c33-cf152733ab27.png)